nodejs-babel-core/nodejs-babel-core.spec

```diff
@@ -166,7 +166,7 @@ done
 %setup -T -q -a 61 -D -n npm_cache
 
 %build
-npm install %{npm_name}@%{version} --cache-min Infinity --cache .
+npm install --cache-min Infinity --cache . --global-style true %{npm_name}@%{version}
```

Comment:
Perhaps but purpose of this PR is rather get COPR rolling rather than fixing all our packages, I'd rather keep it on as we go. That would create unnecessary release version bumps.